static.config.js

```diff
@@ -735,6 +736,19 @@ const makeAllPages = async (langCode, incrementalPageId) => {
           ),
         ),
       );
+    }
+
+    // not all pages have instances (events and locations not under departments)
+    return Promise.all(
+      pageAtUrlInfo.node.janisUrls.map(instanceOfPage =>
```

Comment:
so are we not getting `janisUrls` on pages that have `janisInstances`? Are we building these pages twice and it's just working so we're cool with it? (i'm down)

Reply:
we get janisUrls on the other pages, but they are built at their instances. event & location pages are always built at their janisUrl. 

I was going to say they aren't built twice, but now im thinking that they might be built twice at the same place if they have events and topics...